absl/base/BUILD.bazel

```diff
@@ -38,6 +38,10 @@ cc_library(
         "internal/spinlock_wait.h",
     ],
     copts = ABSL_DEFAULT_COPTS,
+    linkopts = select({
```

Comment:
Let's try to avoid this repeated code everywhere.

https://github.com/abseil/abseil-cpp/blob/master/absl/copts.bzl#L118 has a ABSL_DEFAULT_COPTS var.  Can you add an ABSL_DEFAULT_LINKOPTS var to copts.bzl?

I think you can leave out the specific case of "//absl:freebsd".  Just add both "-lm" and "-pthread" to "//conditions:default".  As far as I know all the POSIX build environments supported by Abseil need both, but we are probably somehow implicitly using "-lm" on Linux.  It should be harmless to add them both as defaults and not special case FreeBSD here.

Reply:
I've now changed to a global linkopts var and dropped the freebsd definition.